lib/resources/r4/device.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: Device
+field_name_base_url: https://www.hl7.org/fhir/device-definitions.html#Device
+fields:
+- name: identifier
+  required: 'No'
+  cardinality: 0..*
+  type: Identifier
+  description: Instance identifier
+  children:
+
+  - name: type
+    required: 'No'
+    cardinality: 0..1
+    type: CodeableConcept
+    description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+    url: http://hl7.org/fhir/r4/datatypes-definitions.html#Identifier.type
+    binding:
+      description: The type used to determine which identifier to use for a specific purpose.
+      note: Currently will return SNO code from the below system.
+      terminology:
+      - display: Identifier Type Codes
+        system: http://hl7.org/fhir/identifier-type
```

Comment:
Is this right or do you guys use:

http://terminology.hl7.org/CodeSystem/v2-0203

I thought device doesnt have mappings?

Reply:
Yes, device doesn't have mappings. I think I forgot to change it. Changed it [here](https://github.com/cerner/fhir.cerner.com/pull/286/commits/4f208c84cb5b9f4f6973dd9983f9d4504d8d001a#diff-7bad88b7f4265915d4ac88a059a5d185R23)

Reply:
+1